lua/lazyvim/plugins/extras/test/core.lua

```diff
@@ -0,0 +1,51 @@
+return {
+  {
+    "nvim-neotest/neotest",
+    dependencies = {
+      -- which key integration
+      {
+        "folke/which-key.nvim",
+        opts = {
+          defaults = {
+            ["<leader>t"] = { name = "+test" },
+          },
+        },
+      },
+    },
+    config = function(_, opts)
+      -- get neotest namespace (api call creates or returns namespace)
+      local neotest_ns = vim.api.nvim_create_namespace("neotest")
+      vim.diagnostic.config({
+        virtual_text = {
+          format = function(diagnostic)
+            local message = diagnostic.message:gsub("\n", " "):gsub("\t", " "):gsub("%s+", " "):gsub("^%s+", "")
+            return message
+          end,
+        },
+      }, neotest_ns)
+      local group = vim.api.nvim_create_augroup("lazyvim_neotest_close_with_q", { clear = true })
+      vim.api.nvim_create_autocmd("FileType", {
+        group = group,
+        pattern = {
+          "neotest-output",
+        },
+        callback = function(event)
+          vim.bo[event.buf].buflisted = false
+          vim.keymap.set("n", "q", "<cmd>close<cr>", { buffer = event.buf, silent = true })
+        end,
+      })
+
+      require("neotest").setup(opts)
+    end,
+    -- stylua: ignore
+    keys = {
```

Comment:
Some of these are clashing with other LazyVim mappings we should probably avoid that? 😅 (leader t t, is for sure one of those)

Reply:
I do not see `<leader>t` anywhere in the keymaps?
There is no `<leader>t+` group anywhere https://github.com/LazyVim/LazyVim/blob/25d37a2cdd6202f1d767595454a7f84f77bbd83e/lua/lazyvim/plugins/editor.lua#L240-L250
Is this something you have in your config maybe?

I see them defined here in your dotfiles 😄 
https://github.com/kevinrobayna/dotfiles/blob/373cc8669849bb77874c9a9b50629e1de089ad91/config/.config/nvim/lua/config/keymaps.lua#L24-L27 https://github.com/kevinrobayna/dotfiles/blob/373cc8669849bb77874c9a9b50629e1de089ad91/config/.config/nvim/lua/plugins/overseer.lua#L5